loader/loader.go

```diff
@@ -571,6 +573,14 @@ func loadShellCommand(value interface{}) (interface{}, error) {
 	return value, nil
 }
 
+func loadHealthcheck(value interface{}) (interface{}, error) {
+	if str, ok := value.(string); ok {
+		words, err := shellwords.Parse(str)
+		return append([]string{"CMD-SHELL"}, words...), err
+	}
+	return append([]string{"CMD"}, value.([]string)...), nil
```

Comment:
It's just 2 line above, it's using `shellwords.Parse` (as `command`) and use `CMD-SHELL` instead of `CMD`.